config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-presubmits-release-0-17.yaml

```diff
@@ -12,7 +12,7 @@ presubmits:
       description: "Verify source code of node-feature-discovery: check formatting, lint etc."
     spec:
       containers:
-      - image: public.ecr.aws/docker/library/golang:1.23
+      - image: public.ecr.aws/docker/library/golang:1.24.0
```

Comment:
Let's not bump in the release-0.17 (at least in this pr)

Reply:
sorry, didn't even realize that it was for the release branch. Fixed.